### PR TITLE
Added possibility to don't log sensitive data with sendkeys method

### PR DESCRIPTION
**Pull request by dhlavac**
Sometimes you don't want to log sensitive data that are filled to inputs. For example, it would be useful when you need to upload a report but omit sensitive data from the logger inside.  PR adds this functionality by adding an option to the `send_keys` method.

### CODE REVIEW

**Review by digitronik — APPROVED** [review]
It makes sense to me.
@mshriver ^^

**Comment by digitronik** [comment]
ahhh I forgot, 
@dhlavac I think we need change in other places as well https://github.com/RedHatQE/widgetastic.core/blob/master/src/widgetastic/widget/input.py#L64

**Review by mshriver — APPROVED** [review]

File: src/widgetastic/ouia/input.py [hunk 1]
```diff
@@ -16,12 +16,17 @@ def value(self):
...
+            sensitive: Bool, If is set to True do not log sensitive data.
+        """
         current_value = self.value
         if value == current_value:
             return False
         # Clear and type everything
         self.browser.click(self)
         self.browser.clear(self)
-        self.browser.send_keys(value, self)
+        self.browser.send_keys(value, self, sensitive)
```
Comment (digitronik) [inline src/widgetastic/ouia/input.py:31]: Need coverage for this line as per codecov

**Review by digitronik — APPROVED** [review]
LGTM :+1: 

needs @mshriver final ack to get in :).

File: src/widgetastic/ouia/input.py (hunk 1 above)
Comment (dhlavac) [reply src/widgetastic/ouia/input.py:31]: Sure I can add it, should it cover only the widget function or also check the logger for masking logs if it is sensitive?

**Comment by dhlavac** [comment]
@mshriver can we merge it ?

**Pull request merged**